src/condor_starter.V6.1/io_proxy_handler.cpp

```diff
@@ -75,7 +75,16 @@ int IOProxyHandler::handle_request( Stream *s )
 	char line[CHIRP_LINE_MAX];
 	ReliSock *r = (ReliSock *) s;
 
-	if(r->get_line_raw(line,CHIRP_LINE_MAX)>0)  {
+	int bytes = r->get_line_raw(line, CHIRP_LINE_MAX);
+	if (bytes >= CHIRP_LINE_MAX) {
```

Comment:
(Batch with above suggestion.)
```suggestion
	int bytes = r->get_line_raw(line, CHIRP_LINE_MAX+1);
	if (bytes > CHIRP_LINE_MAX) {
```